src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -26,28 +29,34 @@ public class Drivetrain extends SubsystemBase {
 
 
   public Drivetrain() {
-
     // TODO 1.1.2: Initialize motors
-
+       leftMotor1 = new CANSparkMax (DriveConstants.LEFT_MOTOR_1_ID, MotorType.kBrushless);
+       rightMotor1 = new CANSparkMax (DriveConstants.RIGHT_MOTOR_1_ID, MotorType.kBrushless);
     // TODO 1.1.3: Set motors to brake mode
-  
+    leftMotor1.setIdleMode(IdleMode.kBrake);
+    rightMotor1.setIdleMode(IdleMode.kBrake);
+
+      
     // TODO 1.1.4: Make motor2s follow motor1s
 
     // TODO 1.2.4: Invert motors if necessary
-
+        leftMotor1.setInverted(true);
+        rightMotor1.setInverted(true);
```

Comment:
I'm still not sure why this part works. Change leftMotor1 to be not inverted for now so it's the same as everyone else's.